src/command_modules/azure-cli-batchai/HISTORY.rst

```diff
@@ -2,6 +2,11 @@
 
 Release History
 ===============
+0.3.1
++++++
+* Fixed `-o table` option for `az batchai cluster node list` and `az batchai job node list` commands.
+* Fixed description and examples for `--setup-task` usage. The setup task is running under root account.
```

Comment:
Don't include changes to help content in the update history.  The first entry is fine though.

Reply:
thanks, removed